desc/magnetic_fields/_core.py

```diff
@@ -252,9 +273,64 @@ def compute_Bnormal(
         B = self.compute_magnetic_field(
             coords, basis="xyz", source_grid=source_grid, params=params
         )
-
         Bnorm = jnp.sum(B * surf_normal, axis=-1)
 
+        if calc_Bplasma:
+            ## do virtual casing to find plasma contribution to Bnormal
+            vc_eval_data = eq.compute(
+                [
+                    "K_vc",
+                    "B",
+                    "R",
+                    "phi",
+                    "Z",
+                    "e^rho",
+                    "n_rho",
+                    "|e_theta x e_zeta|",
+                ],
+                grid=eval_grid,
+            )
+            vc_data = eq.compute(
+                [
+                    "K_vc",
+                    "B",
+                    "R",
+                    "phi",
+                    "Z",
+                    "e^rho",
+                    "n_rho",
+                    "|e_theta x e_zeta|",
+                ],
+                grid=vc_source_grid,
```

Comment:
think we need an actual default for this, otherwise it will use a quadrature grid